packages/editor-ui/src/components/NodeWebhooks.vue

```diff
@@ -12,159 +10,163 @@ import {
-						toggleTitle: this.$locale.baseText('nodeWebhooks.webhookUrls.formTrigger'),
-						clickToDisplay: this.$locale.baseText(
-							'nodeWebhooks.clickToDisplayWebhookUrls.formTrigger',
-						),
-						clickToHide: this.$locale.baseText('nodeWebhooks.clickToHideWebhookUrls.formTrigger'),
-						clickToCopy: this.$locale.baseText('nodeWebhooks.clickToCopyWebhookUrls.formTrigger'),
-						testUrl: this.$locale.baseText('nodeWebhooks.testUrl'),
-						productionUrl: this.$locale.baseText('nodeWebhooks.productionUrl'),
-						copyTitle: this.$locale.baseText('nodeWebhooks.showMessage.title.formTrigger'),
-						copyMessage: this.$locale.baseText('nodeWebhooks.showMessage.message.formTrigger'),
-					};
-
-				default:
-					return {
-						toggleTitle: this.$locale.baseText('nodeWebhooks.webhookUrls'),
-						clickToDisplay: this.$locale.baseText('nodeWebhooks.clickToDisplayWebhookUrls'),
-						clickToHide: this.$locale.baseText('nodeWebhooks.clickToHideWebhookUrls'),
-						clickToCopy: this.$locale.baseText('nodeWebhooks.clickToCopyWebhookUrls'),
-						testUrl: this.$locale.baseText('nodeWebhooks.testUrl'),
-						productionUrl: this.$locale.baseText('nodeWebhooks.productionUrl'),
-						copyTitle: this.$locale.baseText('nodeWebhooks.showMessage.title'),
-						copyMessage: undefined,
-					};
-			}
-		},
-	},
-	watch: {
-		node() {
-			this.isMinimized = !OPEN_URL_PANEL_TRIGGER_NODE_TYPES.includes(this.nodeType.name);
-		},
-	},
-	methods: {
-		copyWebhookUrl(webhookData: IWebhookDescription): void {
-			const webhookUrl = this.getWebhookUrlDisplay(webhookData);
-			void this.clipboard.copy(webhookUrl);
-
-			this.showMessage({
-				title: this.baseText.copyTitle,
-				message: this.baseText.copyMessage,
-				type: 'success',
-			});
-			this.$telemetry.track('User copied webhook URL', {
-				pane: 'parameters',
-				type: `${this.showUrlFor} url`,
-			});
-		},
-		getWebhookUrlDisplay(webhookData: IWebhookDescription): string {
-			if (this.node) {
-				return this.workflowHelpers.getWebhookUrl(
-					webhookData,
-					this.node,
-					this.isProductionOnly ? 'production' : this.showUrlFor,
-				);
-			}
-			return '';
-		},
-		isWebhookMethodVisible(webhook: IWebhookDescription): boolean {
-			try {
-				const method = this.workflowHelpers.getWebhookExpressionValue(webhook, 'httpMethod', false);
-				if (Array.isArray(method) && method.length !== 1) {
-					return false;
-				}
-			} catch (error) {}
-
-			if (typeof webhook.ndvHideMethod === 'string') {
-				return !this.workflowHelpers.getWebhookExpressionValue(webhook, 'ndvHideMethod');
-			}
-
-			return !webhook.ndvHideMethod;
-		},
-
-		getWebhookHttpMethod(webhook: IWebhookDescription): string {
-			const method = this.workflowHelpers.getWebhookExpressionValue(webhook, 'httpMethod', false);
-			if (Array.isArray(method)) {
-				return method[0];
-			}
-			return method;
-		},
+const props = defineProps<{
+	node: INodeUi;
+	nodeTypeDescription: INodeTypeDescription | null;
```

Comment:
```suggestion
	nodeTypeDescription?: INodeTypeDescription
```

wouldn't it be better like this?

Reply:
yes of course :), makes sense